packages/bridge/src/axelar/transfer-status.ts

```diff
@@ -1,27 +1,27 @@
-import { ITxStatusReceiver, ITxStatusSource } from "@osmosis-labs/stores";
+import { poll } from "@osmosis-labs/utils";
 
 import type {
-  BridgeProviderContext,
   BridgeTransferStatus,
+  Environment,
```

Comment:
Thoughts on renaming this to something more descriptive like `BridgeEnvironment`?

Reply:
I can

But since the context of the package is for bridging, wouldn't prepending everything with `Bridge` be redundant? Then the caller, if they choose (or need to clarify their context) can prepend bridge to the type upon import

Reply:
This is a good point. I believe this could improve import autocomplete and enhance code readability. However, I'm fine with whatever you consider is best